API.yaml

```diff
@@ -489,6 +489,92 @@ paths:
             application/json:
               schema:
                 type: string
+ /experiments/{expUUID}/annotations:
+    get:
+      summary: Get the collection of annotations categories for an experiment.
+      operationId: getExperimentAnnotations
+      tags:
+      - Annotations
+      parameters:
+      - name: expUUID
+        in: path
+        description: The UUID of the experiment in the server
+        required: true
+        schema:
+          type : integer
+          format: int128
+      requestBody:
+        description: Query parameters to fetch the annotations.
+          Usually contains these parameters, but not limited to
+          - list of outputIds
+          - startTime
+          - endTime
+        required: false
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Query'
+      responses:
+        200:
+          description: Returns a list of annotation categories for this trace
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Bookmark'
+        404:
+          description: No such experiment
+          content:
+            application/json:
+              schema:
+                type: string
+/experiments/{expUUID}/annotations/{annotationId}:
```

Comment:
I don't really understand this endpoint, but I think what your are trying to do is the get annotation for a specific output. If this is the case I think we should use _/experiments/{expUUID}/outputs/XY/{outputID}/annotations:_ and _/experiments/{expUUID}/outputs/timeGraph/{outputID}/annotations:_ this way you don't have to specify a list of outputIds and having two endpoint gives the opportunity to specify a different model for XY and TimeGraph